packages/react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -97,284 +93,231 @@ export const FocusTrapZone: React.FunctionComponent<IFocusTrapZoneProps> & {
     'data-is-focus-trap-zone-bumper': true,
   } as React.HTMLAttributes<HTMLDivElement>;
 
-  const focus = React.useCallback(() => {
+  const focusElementAsync = React.useCallback((element: HTMLElement) => {
+    if (element !== firstBumper.current && element !== lastBumper.current) {
+      focusAsync(element);
+    }
+  }, []);
+
+  /**
+   * Callback to force focus into FTZ (named to avoid overlap with global focus() callback).
```

Comment:
This one had previously been called `focus()` in the FC conversion, which if things get moved around could result in the proper callback no longer being in scope but still appearing to work due to the global name overlap